libvirt.go

```diff
@@ -343,6 +366,62 @@ func (l *Libvirt) Domains() ([]Domain, error) {
 	return result.Domains, nil
 }
 
+// DomainCreate starts specified domain
+func (l *Libvirt) DomainCreate(dom string) error {
```

Comment:
no need for this duplicate function, `DomainCreateWithFlags` should be `DomainCreate(dom string, flags DomainCreateFlags)` where flags can be `0`.